CMSgov/mint-app
# [EASI-2985] New Solution: Beneficiary Claims Data API (BCDA)

# EASI-2985

## Changes and Description

- Update GQL Enums
- Updated SQL Enums
- Updated Possible Solutions
- Updated Possible Solution - Needs Link
- Boilerplate update to FE solution map (needs more work, timeline, FE review, etc)



## How to test this change

- Run `scripts/dev down && scripts/dev up && scripts/dev db:seed`
- Open MINT at `localhost:3005`
- Open any model plan
- Navigate to `Ops, Eval, and Learning`
- Navigate to Page 3
- Answer that you are sending to the appropriate people
  - I believe the checkbox needed is `Participant-Level Data` but need to confirm
- Navigate to `IT Solutions`
- Navigate to suggested solutions
- Confirm that `BCDA` is a suggested solution


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
